pkg/network/vmispec/defaults_test.go

```diff
@@ -0,0 +1,111 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2024 Red Hat, Inc.
+ *
+ */
+
+package vmispec_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	v1 "kubevirt.io/api/core/v1"
+
+	"kubevirt.io/kubevirt/pkg/libvmi"
+	"kubevirt.io/kubevirt/pkg/network/vmispec"
+)
+
+var _ = Describe("Default pod network", func() {
+	DescribeTable("It should not automatically add the pod network to the VMI", func(vmi *v1.VirtualMachineInstance) {
+		origSpec := vmi.Spec.DeepCopy()
+
+		Expect(vmispec.SetDefaultNetworkInterface(stubClusterConfig{}, &vmi.Spec)).To(Succeed())
+		Expect(vmi.Spec).To(Equal(*origSpec))
```

Comment:
nit: The separation between the action that is checked and the assertion of its result helps the reader to differentiate between the two.
I usually use a newline to express that.

e.g.
```go
Expect(vmispec.SetDefaultNetworkInterface(stubClusterConfig{}, &vmi.Spec)).To(Succeed())

Expect(vmi.Spec).To(Equal(*origSpec))
```

Reply:
Done.